src/pages/staticPages/tools/mutationMapper/MutationMapperToolStore.ts

```diff
@@ -405,4 +441,53 @@ export default class MutationMapperToolStore {
     @action public clearCriticalErrors() {
         this.criticalErrors = [];
     }
+
+    // override fetch function in GenomeNexusCache with selected GN client
```

Comment:
lets avoid this repetition with a factory for making these fetch functions:
`function fetchFactory(fields, client){
  
  return function(query) {
    console.log(fields);
    console.log(client)
    console.log(query);
  }
  
}

var func1 = makeFetch(["one","two"]);

var func2 = makeFetch(["three", "four"]);

func1("query1");

func2("query2");